thumbor/__init__.py

```diff
@@ -10,5 +10,5 @@
 
 '''This is the main module in thumbor'''
 
-__version__ = "6.3.2"
+__version__ = "6.4.0"
```

Comment:
It's not a matter of preference. It's just that doing so would give us a change to include into 6.4.0 any other changes that may be ready to to go, not just this one.